io.github.tntwise.REAL-Video-Enhancer.yml

```
@@ -0,0 +1,39 @@
+app-id: io.github.tntwise.REAL-Video-Enhancer
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: REAL-Video-Enhancer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-videos
+modules:
+  - name: REALVideoEnhancer
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 REAL-Video-Enhancer /app/bin/REAL-Video-Enhancer
+      - mkdir /app/bin/lib/
+      - cp -r lib/* /app/bin/lib/
+      - mkdir /app/bin/PyQt5.uic.widget-plugins/
+      - cp -r PyQt5.uic.widget-plugins/* /app/bin/PyQt5.uic.widget-plugins/
+      - install -D io.github.tntwise.REAL-Video-Enhancer.svg -t /app/share/icons/hicolor/scalable/apps
+      - install -D io.github.tntwise.REAL-Video-Enhancer.appdata.xml -t /app/share/metainfo
+      - install -D io.github.tntwise.REAL-Video-Enhancer.desktop -t /app/sahare/applications
```

```suggestion
      - install -D io.github.tntwise.REAL-Video-Enhancer.desktop -t /app/share/applications
```

that's why the build fails.